.github/workflows/formatting.yaml

```diff
@@ -4,15 +4,23 @@ on: push
 
 jobs:
   clang-format:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
 
     steps:
     - name: Checkout repository
       uses: actions/checkout@v4
 
+    - name: Display clang-format version
+      run: |
+        clang-format --version
+
+    - name: Display clang-format versions
+      run: |
+        ls /usr/bin/clang-format*
+
```

Comment:
I removed the one where it lists all the `clang-format` executables but merged the `clang-format --version` in the `Run clang-format` step. I think it will be useful for debugging when the CI suddenly changes what it's flagging.